pyproject.toml

```diff
@@ -0,0 +1,109 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "itk-fastbilateral"
+version = "0.1.0"
+description = "Faster implementation of the itkBilateralImageFilter. From the insight journal submission: https://www.insight-journal.org/browse/publication/692."
+readme = "README.rst"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Pablo Hernandez Cerdan", email = "pablo.hernandez.cerdan@outlook.com" },
```

Comment:
The original IJ article author, Jordan Woehr, should be also listed.

Reply:
PR #6.